build-openresty.sh

```diff
@@ -29,14 +29,28 @@ mkdir -p /tmp/build/usr/local/kong
 mkdir -p /tmp/build/
 mkdir -p /work
 
-if [ "$DEBUG" == 1 ]
+if [ -z "$KONG_NGINX_MODULE" ]
 then
-  KONG_NGX_BUILD_ARGS="--debug"
+  KONG_NGINX_MODULE="master"
 fi
 
-if [ -z "$KONG_NGINX_MODULE" ]
+NGX_BUILD_FLAGS=(
+  "--openresty $RESTY_VERSION"
+  "--openssl $RESTY_OPENSSL_VERSION"
+  "--luarocks $RESTY_LUAROCKS_VERSION"
+  "--kong-nginx-module $KONG_NGINX_MODULE"
+  "--pcre $RESTY_PCRE_VERSION"
+  "--work /work"
+)
+
+if [ ! -z "$OPENTRACING_VERSION" ]
 then
-  KONG_NGINX_MODULE="master"
+  NGX_BUILD_FLAGS+=("--opentracing $OPENTRACING_VERSION")
+fi
+
+if [ "$DEBUG" == 1 ]
```

Comment:
Shuffling this around such that it's more obvious the `--debug` flag isn't being changed would be desirable.